README.md

```diff
@@ -6,24 +6,66 @@ For Angular 1 see [ng-redux](https://github.com/wbuchwalter/ng-redux)
 [![Circle CI](https://circleci.com/gh/angular-redux/ng2-redux/tree/master.svg?style=svg)](https://circleci.com/gh/angular-redux/ng2-redux/tree/master)
 [![npm version](https://img.shields.io/npm/v/ng2-redux.svg?style=flat-square)](https://www.npmjs.com/package/ng2-redux)
 
-`ng2-redux` lets you easily connect your Angular 2 components with Redux.
+## Why use Redux?
+
+Redux is a predictable state container for complex
+web applications. It uses one-way data-flow and global
+immutable state to make your application easier to reason
+about.
+
+As a mature and established library, Redux also has a
+large community of developer tools and middleware that can help simplify your application
+even further.
+
+## Why use Ng2-Redux?
+
+Redux itself is framework-agnostic, so you need a small extra layer (a
+'binding') to integrate it with a particular framework.
+[react-redux](https://npmjs.com/package/react-redux) fullfils this role for React.
+Ng2-Redux makes Redux work with Angular 2.
+
+Ng2-Redux provides several advantages to the Angular 2 developer:
+
+1. It helps you to use Redux in your Angular 2 application!
+2. It plays well with the Angular 2 dependency injector.
+3. It has first-class support for a TypeScript-based workflow, using
+  the official Redux typings.
+4. It exposes state as RxJS Observables for
+  better integration with Angular 2's API.
+5. It cuts down the boilerplate often associated
+  with Redux using TypeScript decorators for a simple,
+  declarative experience.
+
+Note that this is not a reimplementation of the Redux
+pattern. _Ng2-Redux helps you to use the actual
+Redux in an Angular 2 application_. This means
+that it's completely compatible with the tools provided
+by the vibrant Redux community, including:
+
+* Middleware like [redux-saga](https://npmjs.com/package/redux-saga),
+[redux-thunk](https://npmjs.com/package/redux-thunk),
+and [redux-logger](https://npmjs.com/package/redux-logger).
+* The official Chrome extension Redux dev tools.
+(https://github.com/zalmoxisus/redux-devtools-extension)
+* Community store enhancers.
 
 ## Table of Contents
 
 - [Installation](#installation)
 - [Quick Start](#quick-start)
 - [Usage](#usage)
   - [The decorator interface](#the-decorator-interface)
-    - [The @dispatch decorator](#the-dispatch-decorator)
     - [The @select decorator](#the-select-decorator)
+    - [The @dispatch decorator](#the-dispatch-decorator)
```

Comment:
To me "select" logically comes before "dispatch", since your component would usually first select some data from the store for display, then dispatch an action.